src/Resources/doc/middleware.md

```diff
@@ -41,7 +41,7 @@ csa_guzzle:
             middleware: ['!my_middleware']
 ```
 
-Disabled middleware have priority over enabled middleware.
+Warning: you cannot use the "disable" feature along with the "enable" feature.
```

Comment:
I would change the wording as "You can either whitelist or blacklist middleware. Using both whitelisting and blacklisting will trigger an exception."